acct-group/ollama/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>zdanevich.vitaly@ya.ru</email>
+		<name>Vitaly Zdanevich</name>
+		<description>A copy from https://github.com/MrPenguin07/ebuilds/tree/master/dev-ml/ollama</description>
+	</maintainer>
+	<use>
+		<flag name="nvidia">Add support of nvidia</flag>
+		<flag name="amd">Add support of amd</flag>
+	</use>
```

Comment:
This `use` is for the ebuild, not for acct-group, right?

Reply:
Thanks I'll sort it out rn

Reply:
Should i change the description or no?

Reply:
Actually I'll drop the description, I took a look at the other acct-user/group packages they didn't use a description tag